Overview.bs

```diff
@@ -445,7 +451,7 @@ of the patch map table. A map is encoded in one of two formats:
   </tr>
   <tr>
     <td>uint32</td>
-    <td><dfn for="Format 1 Patch Map">id</dfn>[4]</td>
+    <td><dfn for="Format 1 Patch Map">compatibilityId</dfn>[4]</td>
```

Comment:
It's hard to read with the markup, but it's still 128 bits there's a [4] after indicating it's an array of 4 uint32's. Agreed, we don't actually talk about how to set this anywhere and should definitely provide some guidance.

Reply:
Should be a smallish change so I'll add that to this PR.

Reply:
I should add: Maybe even more important than preventing the attempted integration of a patch from a different font, this field was originally intended to prevent attempted integration of a patch from a *different encoding* of the *same* font. I've run into that in practice a number of times even with my very limited use of the system, and having the ID match fail is far preferable to spending an hour wondering if the system has a bug before realizing that the files don't match. It can happen when you overwrite one file but not another, and it can also happen because of caches that don't get cleared.

Reply:
Oh, my bad. (I wrote that second comment before I saw yours -- github glitch).

Reply:
Sorry, my head has been fuzzy for the past few days. Normally I'm better at not going on these mistaken detours.

Reply:
I think I picked 128 because it was the number where, assuming you have a decent random number generator with a real seed, you don't have to worry about manually checking for collisions anymore. It might be worth putting something to that effect in a "Note".

Reply:
Done.

Reply:
Added a bit of extra text to avoid collisions within a particular IFT font.